db/migrate/20180129081557_add_digest_run_subscriber_id_to_subscription_contents.rb

```diff
@@ -0,0 +1,6 @@
+class AddDigestRunSubscriberIdToSubscriptionContents < ActiveRecord::Migration[5.1]
+  def change
+    add_column :subscription_contents, :digest_run_subscriber_id, :integer
+    add_index :subscription_contents, :digest_run_subscriber_id
```

Comment:
Similar comment about foreign keys, I think we have those on other columns in this model

Reply:
I've made this a foreign key. Not sure we want the dependent destroy on this one. What do you think?